Rakefile

```diff
@@ -0,0 +1,20 @@
+# encoding: UTF-8
+
+$:.unshift File.join(File.dirname(__FILE__), 'lib')
+
+require 'rubygems' unless ENV['NO_RUBYGEMS']
```

Comment:
Actually I'm not sure... it's something I saw a bunch of other gems doing, so I copied and pasted :/